sktime/forecasting/naive_variance.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+
+"""Implements naive variance functionality to tune forecasters."""
+
+__author__ = ["IlyasMoutawwakil"]
+
+
+import numpy as np
+import pandas as pd
+from scipy.stats import norm
+from sklearn.base import clone
+
+from sktime.forecasting.base import BaseForecaster
+from sktime.utils.validation.forecasting import check_alpha, check_fh
+
+
+class NaiveVariance(BaseForecaster):
+    """
+    Compute the prediction variance using the 'simple strategy'.
+
+    Parameters
+    ----------
+    fh : int, list, np.array or ForecastingHorizon
+        Forecasting horizon
+    X : pd.DataFrame, optional (default=None)
+        Exogenous time series
+    cov : bool, optional (default=False)
+        Wether to return marginal variances or covariance matrices.
+    """
+
+    _tags = {
+        "handles-missing-data": False,
+        "ignores-exogeneous-X": True,
+        "capability:pred_int": True,
+        # deprecated and will be renamed to capability:predict_quantiles in 0.11.0
+        "capability:pred_var": True,
+        # deprecated and will be renamed to capability:predict_variance in 0.11.0
+    }
+
+    def __init__(self, forecaster):
+
+        self.forecaster = forecaster
```

Comment:
component forecasters should be cloned, or the tests will fail!

Reply:
Makes sense, it's cloned now.